variables.tf

```diff
@@ -27,7 +29,9 @@ variable "wait_for_cluster_cmd" {
   default     = "until curl -k -s $ENDPOINT/healthz >/dev/null; do sleep 4; done"
 }
 
-# Worker Nodes
+// ----------------------------------------------------------------------------
+// Worker Nodes Variables
+// ----------------------------------------------------------------------------
 variable "desired_number_of_nodes" {
   description = "The number of worker nodes to use for the cluster. Defaults to 3"
```

Comment:
I don't think you need to add 'Defaults to 3' to the. description. That's given and documented implicitly.

Reply:
Same for max.